web/html/src/utils/data-providers/page-control.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+declare var userPrefPageSize: number;
```

Comment:
Again the ugly global... :nauseated_face:

Reply:
Now I'm using the mixin provided from jade :+1: